main.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
for the files in the root, the package should be "game" or w/e your package is called (unlike under /cmd/, where the package is always "main")